crates/tabby/src/serve/mod.rs

```diff
@@ -161,8 +160,20 @@ pub async fn main(config: &Config, args: &ServeArgs) {
     }
 
     info!("Starting server, this might takes a few minutes...");
+
+    // swagger add local server
+    let mut doc = ApiDoc::openapi();
```

Comment:
extract this into a function like `add_localhost_server(doc: Doc) -> Doc`

Reply:
👌，Let me refactor the code ～